src/psyclone/tests/test_files/dynamo0p3/infrastructure/domain_size_config_mod.f90

```diff
@@ -121,8 +121,6 @@ subroutine read_namelist( file_unit, local_rank )
     buffer_real_r_def(3) = planar_domain_min_x
     buffer_real_r_def(4) = planar_domain_min_y
```

Comment:
Reading in namelists in LFRic also works with MPI broadcast. Here it was compiled but was not run as this stripped-down infrastructure library does not support MPI. I disabled the MPI broadcast call because I actually need to run this config file to read a namelist in one of my tutorial examples.